client/my-sites/plugins/plugin.jsx

```diff
@@ -113,9 +116,8 @@ export default React.createClass( {
 
 	refreshSitesAndPlugins( nextProps ) {
 		this.setState( this.getSitesPlugin( nextProps ) );
-
 		// setTimeout to avoid React dispatch conflicts.
-		setTimeout( () => {
+		this.pluginRefreshTimeout = setTimeout( () => {
```

Comment:
`setTitle` has its own `setTimeout`: https://github.com/Automattic/wp-calypso/blob/a7478174b0441760c332fcb4b8c454f5b8d73dfc/shared/lib/screen-title/actions.js#L16, do we know why that one isn't enough?